releasenotes/notes/deprecate-post-install-webhook.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: deprecate-istioctl-cmd
```

Comment:
`kind` should be any of these. Maybe you can just add `kind: feature` and in the message, we should say `**Removed**`

Which means that the specific `feature is removed`.

https://github.com/istio/istio/blob/5ea614d48ee3887884922789985f6cef88346ecf/releasenotes/template.yaml#L6-L13